addons/l10n_ar_withholding/models/account_tax.py

```diff
@@ -7,13 +7,36 @@ class AccountTax(models.Model):
     _inherit = 'account.tax'
 
     l10n_ar_withholding_payment_type = fields.Selection(
-        [('supplier', 'Supplier'), ('customer', 'Customer')], 'Argentinean Withholding type',
-        compute="_compute_l10n_ar_withholding_payment_type", store=True, readonly=False)
-
+        [('supplier', 'Supplier'), ('customer', 'Customer')], 'Argentinean Withholding Type',
```

Comment:
@jco-odoo I keep as it is now because we follow this approach https://github.com/odoo/odoo/blob/808c5378e6a304e684ded94f9390138409cbf7e6/addons/account/models/account_payment.py#L93 and also we use this domain https://github.com/odoo/odoo/blob/755b81956a06bdcc82aa62b1b9053f8768d20c44/addons/l10n_ar_withholding/wizards/l10n_ar_payment_register_withholding.py#L23

Reply:
At least put Vendor Payment in the text then (to be similar to account_payment.py